sample/src/main/java/dev/arkbuilders/sample/storage/StorageDemoFragment.kt

```diff
@@ -105,13 +115,14 @@ class StorageDemoFragment: DialogFragment() {
         }
     }
 
-    private fun refreshMap() {
-        if (map.isEmpty()) {
+    private fun refreshStorage() {
+        if (storage == null) {
             binding.tvMapValues.text = getString(R.string.empty_map)
             return
         }
+        storage!!.writeFS()
```

Comment:
I should check with the crate implementation, but do we really need flushing? That isn't handled by the crate?

Reply:
`storage.set` and `storage.remove` do not write to fs, only `storage.erase` deletes file